backend/src/config/serverConfig.ts

```diff
@@ -35,7 +35,7 @@ class Server {
 
   private middleware() {
     this.app.use(express.json());
-    this.app.use(cors());
+    this.app.use(cors({origin: (process.env.FRONTEND_URL as string)}));
```

Comment:
```suggestion
    this.app.use(cors({origin: (process.env.FRONTEND_URL ? process.env.FRONTEND_URL : "*")}));
```

in this case it's working if the FRONTEND_URL is not set.

What do you think. 
And a warning console log to say that you should set the FRONTEND_URL in the .env

Reply:
I think allowing all origins would pause a security risk, especially when we're using JWT with localStorage, I would rather we force that someone specifies an origin, that's why we're using the `cors` middleware after all

Reply:
OK in this case log a warning to say you have to setup the FRONTEND_URL or just throw an error.

Reply:
I'm not sure I know how to implement that. Where would the warning be logged? If it's a problem we can just implement your simple fix.

Reply:
I will show you. 
Let create a fix for it later